### PR TITLE
#162163248 enable like of a specific comment

### PR DESCRIPTION
**Pull request by SokoPaulSokool**
### What does this PR do?
Enable user like a specific comment

### Description of tasks to be completed?
Show article read time both on viewing all articles and viewing a specific article

### How can this be manually tested?
Add comments to an article
Click the thumbs up button
Number of likes increases after clicking

### Screen shots
image
image

### CODE REVIEW

**Comment by JaymesKat** [comment]
@SokoPaulSokool correct typo in PR description under the section `What does this PR do?`

File: src/components/Comments/CommentEdit/CommentEdit.jsx [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+  clickLike,
+  clickDisLike,
+  clickReply,
+  clickDelete,
+  clickEdit,
+  id
+}) => {
   const handleKeyUp = () => {};
-  
+  let disl = 'text-primary m-1';
```
Comment (JaymesKat) [inline src/components/Comments/CommentEdit/CommentEdit.jsx]: preferably rename this variable `dislikeClass`

File: src/components/Comments/CommentsList/CommentsList.jsx [hunk 2]
```diff
@@ -52,19 +56,30 @@ export class CommentsList extends React.Component {
...
       this.setState({ showComBox: dnone, isEdit: status1 });
     } else {
       this.setState({ showComBox: status2, isEdit: status1 });
     }
-  }
+  };
+
+  clickLike = id => {
+    console.log('Liked');
+    const { CommentLikeAction, slug } = this.props;
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/Comments/CommentsList/CommentsList.jsx:83]: Fix this @SokoPaulSokool

File: src/components/Comments/CommentsList/CommentsList.jsx [hunk 3]
```diff
@@ -52,19 +56,30 @@ export class CommentsList extends React.Component {
...
+    CommentLikeAction(id, slug);
+  };
+  clickDisLike = id => {
+    console.log('disLiked');
+    const { CommentDislikeAction, slug } = this.props;
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/Comments/CommentsList/CommentsList.jsx:89]: Fix this @SokoPaulSokool

File: src/components/Comments/CommentsList/LoadReply.jsx [hunk 4]
```diff
@@ -28,7 +28,9 @@ export const LoadReply = ({
   showComBox,
   art_slug,
   username,
-  img
+  img,
+  clickLike,
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/Comments/CommentsList/LoadReply.jsx:31]: fix this @SokoPaulSokool

File: src/components/Comments/ReplyList/LoadNoReply.jsx [hunk 5]
```diff
@@ -17,9 +16,10 @@ const LoadNoReply = ({
   childId,
   showComBox,
   username,
-  img
+  img,
+  clickLike,
```
[the remark this answers is not in the document]
Comment (JaymesKat) [reply src/components/Comments/ReplyList/LoadNoReply.jsx:19]: Fix this too @SokoPaulSokool

File: src/components/Comments/CommentEdit/CommentEdit.jsx (hunk 1 above)
Comment (SokoPaulSokool) [reply src/components/Comments/CommentEdit/CommentEdit.jsx]: ok let me fix it

**Review by bibangamba — APPROVED** [review]
Great work! I especially like the functionality that only shows a dislike button only after you've liked the comment. This is a smart solution for one of our backend bugs.

**Review by Nta1e — APPROVED** [review]
cool stuff🤝

**Review by JaymesKat — APPROVED** [review]
Good work

**Pull request merged**